src/main/java/be/seeseemelk/mockbukkit/PlayerList.java

```diff
@@ -99,16 +99,16 @@ public boolean isSomeoneOnline()
 	public List<Player> matchPlayer(@NotNull String name)
 	{
 		return onlinePlayers.stream().filter(
-				player -> player.getName().toLowerCase(Locale.ENGLISH).startsWith(name.toLowerCase(Locale.ENGLISH)))
-				.collect(Collectors.toList());
+		           player -> player.getName().toLowerCase(Locale.ENGLISH).startsWith(name.toLowerCase(Locale.ENGLISH)))
+		       .collect(Collectors.toList());
```

Comment:
Perhaps this one can be formatted in the same way as `getPlayerExact` right underneath here.